flutter_screen_recording_web/lib/flutter_screen_recording_web.dart

```diff
@@ -21,6 +21,8 @@ class WebFlutterScreenRecording extends FlutterScreenRecordingPlatform {
   String mimeType;
   Completer<String> _onStopCompleter;
   Function(String result) _onStopCallback;
+  MediaStream _userMediaStream;
```

Comment:
While only getAudio so i think should name it `_userAudioStream`

Reply:
I think _userMediaStream is ok, because  MediaStream contain both video and audio